README.md

```diff
@@ -1,2 +1,37 @@
-# pdf-redaction-annotation-csharp
-This repository contains various examples to use the PDF redaction annotation
+# PDFRedactionAnnotation in C#
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows removing the sensitive text and images from the PDF document using its PDF Redaction feature. This repository contains various examples to use the PDF redaction annotation.
```

Comment:
The [Syncfusion .NET PDF library]() provides an easy way to permanently remove confidential or sensitive information from PDF documents. This repository contains examples of redaction feature to safely, permanently remove confidential information from a PDF document.

Reply:
changed the description.